### DISCUSSION TITLE
Argo Rollouts Controller HPA

### DISCUSSION THREAD

**Discussion by vbrinza** [argoproj/argo-helm#2298] [Ideas]
We have situations when we are doing changes in tens of Argo Rollouts simultaneously which causes the Controller to OOM.
I see the Argo Rollouts Controller chart does not have an HPA template.
Wanted to hear the community/maintainers opinions on introducing one.

**Comment by yu-croco** [comment]
argo-helm follows [upstream](https://github.com/argoproj/argo-rollouts)'s manifest, so I think you get more info there.
Not all resources are available for HPA, so I think it's better to ask about specifications at the upstream.
